.github/workflows/main.yml

```diff
@@ -552,8 +544,18 @@ jobs:
           wget ${{ needs.build-binaries.outputs.public-url-macos }} -O taq-macos
           wget ${{ needs.vscode-extension-workflow.outputs.vsixGcsUrl }}
 
-      - name: Release
-        id: release
+      - name: Pre-release
+        id: pre-release
+        uses: softprops/action-gh-release@v1
+        with:
+          files: |
+            ./release/*
+          draft: false
+          prerelease: true
+
+      - name: Full release
```

Comment:
Could be just me but from an outsider perspective the term 'full release' is a bit misleading as it's making me think that other release types include fewer functionalities or features. 

Unless this is already the terminology the team uses internally I think 'final release` might be more appropriate here

Reply:
I agree with that and I like the term 'final release' as well. There is no consensus that I know of on the term 'full release' so I will make that change.